webpack/components/AnsibleHostDetail/components/JobsTab/RecurringJobsTable.js

```diff
@@ -35,6 +35,19 @@ const RecurringJobsTable = ({ jobs, resourceName, resourceId }) => {
     );
   };
 
+  const actionItems = job => {
+    const items = [];
+    if (job.recurringLogic.meta.canEdit) {
+      items.push({
+        title: __('Cancel'),
+        onClick: onJobCancel(job.recurringLogic.id),
```

Comment:
doesn't it suppose to run on click? 
```suggestion
        onClick: () => onJobCancel(job.recurringLogic.id),
```

Reply:
It runs on click, see l.27.